src/main/java/org/gridsuite/study/server/NetworkModificationService.java

```diff
@@ -481,4 +481,5 @@ public List<EquipmentModificationInfos> splitLineWithVoltageLevel(UUID studyUuid
 
         return result;
     }
+
```

Comment:
Remove new line to avoid useless file modification ?

Reply:
Done